module-apps/apps-common/widgets/AbstractProgressTime.hpp

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2017-2024, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#pragma once
+
+#include <cstdint>
+
+namespace gui
+{
+    namespace progress
+    {
+        using namespace std::chrono_literals;
+        constexpr auto secondsInMinute = std::chrono::seconds(1min).count();
```

Comment:
You can use `utils::time::secondsInMinute` constant from `module_utils\time\time\time_constants.hpp` :wink: If you prefer to leave it as it is, please add `inline` to the declaration.

Reply:
Thanks! I decided to use this one from `utils::time::secondsInMinute` so it's done ;)